app/Models/Game.php

```diff
@@ -18,7 +18,8 @@ class Game extends Model
         'release_year',
         'playing_time_minutes',
         'min_number_players',
-        'max_number_playeres'];
+        'max_number_playeres',
```

Comment:
Maybe there is a reason it is like this, but I notice that "players" seems to be misspelled here!

Reply:
Nope, this is just a typo. Good eyes :)